automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -17,104 +15,19 @@ import org.scalatest.{FreeSpec, ParallelTestExecution}
 
 import scala.concurrent.duration._
 
+/**
+  * This spec verifies cluster status transitions like pause/resume and cluster PATCH.
+  * It is similar in intent to ClusterStatusTransitionsSpec but uses notebooks for validation,
+  * so lives in the notebooks sub-package.
+  */
 class NotebookClusterMonitoringSpec extends FreeSpec with NotebookTestUtils with ParallelTestExecution with BillingFixtures {
 
-  "Leonardo clusters" - {
-
-    // default PetClusterServiceAccountProvider edition
-    "should create a cluster in a different billing project using PetClusterServiceAccountProvider and put the pet's credentials on the cluster" in {
```

Comment:
This spec doesn't need a brand new cluster -- moved to `NotebookPyKernelSpec` (because it runs python to validate the credentials).